appinventor/components-ios/src/File.swift

```diff
@@ -134,6 +134,134 @@ open class File: NonvisibleComponent {
     }
   }
   
+  // Copy the contents from the first file to the second file.
+  func copyFile(fromPath: String, toPath: String) -> Bool {
```

Comment:
NB: For any methods that correspond to blocks, the following rules must be adhered to:

1. Annotate the function (or variable, for properties) with `@objc`
2. Match the name to the block, including case (i.e., `CopyFile` not `copyFile`)
3. Use unlabeled parameters to prevent name mangling

So for this function the declaration should be:

```
@objc func CopyFile(_ fromPath: String, _ toPath: String) -> Bool
```

Reply:
Note that this applies to all of the functions added to this class.